modules/Authentication.php

```diff
@@ -10,17 +10,39 @@ public function __construct($db)
     {
         $this->db = $db;
     }
-
-    // Create instance with session token
-    public static function withSessionToken($db, $session_token)
+    /**
```

Comment:
We can remove the "@" in the comments. Lets follow this pattern https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md as suggested by @prathmeshranaut